src/pages/hero/Hero.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Link as LinkToID } from "react-scroll";
+import StarRatings from "react-star-ratings";
+import { Footer, Navbar } from "../../components/components-index.js";
+import "./Hero.css";
+
+export const Hero = (props) => {
+  const { productData, offerData } = props;
+  const [slider, setSlider] = useState("slide1");
+
+  useEffect(() => {
+    const imgArr = ["slide1", "slide2", "slide3"];
```

Comment:
Suggestion: naming could be more descriptive like `imageArray` or `sliderImageArray`.
Descriptive variable names >> Small variable names

Reply:
Thanks for suggestion! changes done!!